src/components/selection_input/selection_input.ts

```diff
@@ -108,9 +108,9 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
     const ranges = existingSelectionRange.length
       ? existingSelectionRange
       : this.props.ranges
-      ? this.props.ranges.map((xc, i) => ({
+      ? this.props.ranges.map((xc, id) => ({
           xc,
-          id: i.toString(),
+          id,
```

Comment:
@LucasLefevre wdyt of changing the id to number ?

Reply:
LGTM :+1: